Readme update (#564)
rudderlabs/rudder-sdk-js: README.md

# [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#rudderstack-javascript-sdk)RudderStack JavaScript SDK

> The JavaScript SDK lets you track customer event data from your websites and send it to your specified destinations via RudderStack.

Author: ```suggestion
The JavaScript SDK lets you track customer event data from your website and send it to your specified destinations via RudderStack.
```

> The RudderStack JavaScript SDK lets you track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.

Author: ```suggestion
The JavaScript SDK lets you track customer event data from your websites and send it to your specified destinations via RudderStack.
```

> The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.

Reviewer: The first sentence here doesn't make any sense to me. We should rather just say this,

```
The RudderStack JavaScript SDK helps you to track and send user events...
```

Reviewer: As there is no such thing as `analytics.js` or `rudder-analytics.js` library. It's just a source code file in this repository and contributes to the final SDK.

Author: got it - thanks @saikumarrs - I've made the necessary update.

> The [**RudderStack**](https://rudderstack.com/) JavaScript SDK leverages the `analytics.js` library to track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.

Author: ```suggestion
The RudderStack JavaScript SDK lets you track and send user events from your website to RudderStack. You can then further transform and route this event data to the destination platform of your choice.
```
